data/en/listfilter.json

```diff
@@ -1,42 +1,59 @@
 {
-	"name":"listFilter",
-	"type":"function",
-	"syntax":"listFilter(list, function(listElement, [list]) )",
-	"member":"someList.filter(function(listElement, [list]) )",
-	"returns":"string",
-	"related":["listGetAt"],
-	"description":"Used to filter an list to items for which the closure function returns true.",
+	"name": "listFilter",
+	"type": "function",
+	"syntax": "listFilter(list, function(listElement, [list]) )",
+	"member": "someList.filter(function(listElement, [list]) )",
```

Comment:
For some reason the member function is `string.listFilter()`

Reply:
I will change that member function as well as make an example for it

Reply:
Ok - you can create another PR to make those changes. I'll merge this one.